bin.src/addSeds.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import argparse
+import warnings
+import numpy as np
+from lsst.sims.movingObjects import Orbits
+from itertools import repeat
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Add SEDs to a series of orbits.")
+    parser.add_argument("--orbitFile", type=str, default=None,
```

Comment:
Will this script complain if you pass it something that does not have the correct format?

I feel compelled to confess that I am still not conversant in the several different forms of data file used to specify asteroid orbits.  For example, I have the raw *.des files from our MBA models.  The header looks like

!! name      otype    q            ecc       incl      node     peri       T_peri        H     G    epoch         tmp    model

Is this something that this script can work with?